app/services/licences/fetch-customer-contacts.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+/**
+ * Fetches all return logs for a licence which is needed for the view '/licences/{id}/contact-details` page
+ * @module FetchCustomerContactDetailsService
+ */
+
+const { db } = require('../../../db/db.js')
+
+/**
+ * Fetches all contact details for a licence which is needed for the view '/licences/{id}/contact-details` page
```

Comment:
NIT - just being verbose about what we are fetching

```suggestion
 * Fetches all customer contacts for a licence which is needed for the view '/licences/{id}/contact-details` page
```